Project.toml

```diff
@@ -20,6 +20,7 @@ ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+StaticArraysCore = "1e83bf80-4336-4d27-bf5d-d5a4f845583c"
```

Comment:
since we already have `StaticArrays`, can we grab `SVector` from there? I know this doesn't increase our total dependency load, but it's another thing to track.

Reply:
filed https://github.com/ericphanson/ExplicitImports.jl/issues/24 fyi. I think though if you do `using StaticArrays: SVector`, then it won't complain about the implicit import. But when it makes suggestions, it always does the "defining" module.